app/models/channel.rb

```diff
@@ -271,6 +274,25 @@ def register_channel_for_promo
     Promo::RegisterChannelForPromoJob.perform_now(channel_id: id, attempt_count: 0)
   end
 
+  def update_site_banner_lookup!(skip_site_banner_info_lookup: false)
+    site_banner_lookup = SiteBannerLookup.find_or_initialize_by(
+      channel_identifier: details&.channel_identifier || details.brave_publisher_id,
+    )
+    site_banner_lookup.set_sha2_base16
+    site_banner_lookup.set_wallet_status(publisher: publisher)
+    site_banner_lookup.derived_site_banner_info =
+      if skip_site_banner_info_lookup
+        {}
+      else
+        site_banner&.non_default_properties || publisher&.default_site_banner&.non_default_properties || {}
+      end
+    site_banner_lookup.update(
```

Comment:
```suggestion
    site_banner_lookup.update!(
```

We probably want this to raise an exception so the rest of the data doesn't save if this fails?